kong/clustering/compat/removed_fields.lua

```diff
@@ -228,4 +228,11 @@ return {
       "queue.concurrency_limit",
     },
   },
+
+  -- Any dataplane older than 3.9.0
+  [3009000000] = {
```

Comment:
```suggestion
  [30010000000] = {
```
3.9.0 is in code freeze, should we target this to 3.10 or 3.9.1?
Note: I'm not sure that is how we target 3.10, is there a risk we target 3.1.0?